approval_polls/templates/approval_polls/edit.html

```diff
@@ -90,6 +91,12 @@ <h5 style="color:#8B0000;"><b>Please ensure you've entered all choices correctly
         </div>
       {% endfor %}
       {% endwith %}
+      <div id='tag-input'>
+        <h6>(Enter a comma after each tag:)</h6>
+        <input id='allTags' value='{{all_tags}}' type="hidden">
+        <input type='text' name='token-tags' class='form-control' id='tokenTagField' placeholder='Enter keywords' />
```

Comment:
Could you change `keywords` to `tags` here just to keep it consistent for the users ?